components/brave_wallet/browser/keyring_service.cc

```diff
@@ -754,8 +717,14 @@ void KeyringService::RestoreWallet(const std::string& mnemonic,
                                    const std::string& password,
                                    bool is_legacy_brave_wallet,
                                    RestoreWalletCallback callback) {
+  // Added 08.08.2022
+  MaybeMigratePBKDF2Iterations(password);
```

Comment:
Oh this was odd until I expanded the function it looked like this code was being called in the CreateWallet function. It actually is happening in the Restore wallet function so this looks right now.